Paws4Me/HomeCollectionViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  HomeCollectionViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/04/12.
+//
+
+import Foundation
+import UIKit
+
+class HomeCollectionViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
```

Comment:
Conform to `UICollectionViewDataSource` and `UICollectionViewDelegate` in an `extension` on `HomeCollectionViewController`

Reply:
Will do so 👍🏻